_data/simple-icons.json

```
@@ -8077,6 +8077,11 @@
             "hex": "5586A4",
             "source": "https://www.khronos.org/legal/trademarks/"
         },
+        {
+            "title": "openHAB",
+            "hex": "000000",
```

```suggestion
            "hex": "E64A19",
```
Pick dominant color from logo